Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/BoxManipulatorRequestBus.h

```diff
@@ -26,6 +26,10 @@ namespace AzToolsFramework
         virtual AZ::Vector3 GetDimensions() = 0;
         //! Set the X/Y/Z dimensions of the box shape/collider.
         virtual void SetDimensions(const AZ::Vector3& dimensions) = 0;
+        //! Get the translation offset of the box relative to the entity position.
+        virtual AZ::Vector3 GetTranslationOffset() = 0;
```

Comment:
Nit: This could be `const`

Reply:
I thought about that, but the rest of the bus doesn't follow that pattern.

Reply:
Could add them all now 😛 (or maybe as a follow-up 🙂)